packages/cosmic-swingset/lib/ag-solo/vats/lib-wallet.js

```diff
@@ -79,6 +78,41 @@ export async function makeWallet(
     inboxStateChangeHandler(getInboxState());
   }
 
+  // handle the update, which has already resolved to a record. If the offer is
+  // 'done', mark the offer 'complete', otherwise resubscribe to the notifier.
+  function updateOrResubscribe(id, offerHandle, update) {
+    const { updateHandle, done } = update;
+    if (done) {
+      // TODO do we still need these?
+      idToOfferHandle.delete(id);
+
+      const offer = idToOffer.get(id);
+      const completedOffer = {
+        ...offer,
+        status: 'complete',
```

Comment:
Status is already reserved for the "proposed offer" status (did the user reject, accept, cancel, the proposed offer). I think we need a different property here, like "zoeOfferStatus" (I don't like the name, but at least it's clear).

Reply:
Chris convinced me this was the right way. Resolving